src/style.css

```
@@ -0,0 +1,90 @@
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.bigSearch {
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+  margin-top: 12px;
+  margin-bottom: 20px;
+  padding-left: 62px;
+
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+  background: #fff;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19px;
+  background-position: 25px 24px;
+
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.bigSearch:hover {

```

use kebab or snake case
```suggestion
.big-search:hover {
```
```suggestion
.big_search:hover {
```

we don't use camelCase in CSS